packages/ethereum-contracts/.solcover.js

```diff
@@ -5,6 +5,11 @@ module.exports = {
     skipFiles: [
         "mocks/",
         "apps/SuperAppBase.sol",
+        "utils/SuperfluidFrameworkDeployer",
```

Comment:
Is SuperfluidFrameworkDeployer really a good candidate to skip?

Same goes for SuperTokenDeployer

Reply:
It depends, we don't have an explicit definition of what files should be included in our coverage.
In my opinion, we should reduce noise as much as possible and it should solely be the core contracts which are deployed in production.
These are both peripheral contracts that are not used in production and we do already have tests for them.

Reply:
I thought they are used in some of the test cases, hence they should have been covered. I guess my assumption was wrong?